src/main/kotlin/com/toasttab/pulseman/AppStrings.kt

```diff
@@ -0,0 +1,109 @@
+    const val ADD_JAR = "Add Jar"
+    const val ADD_TAB = "Add tab"
+    const val AUTH = "Auth"
+    const val AUTH_PARAMETERS = "Auth parameters"
+    const val AUTH_SETTINGS = "Auth Settings"
+    const val CLASS = "Class"
+    const val CLEAR = "Clear"
+    const val CLEARED_HISTORY = "Cleared history"
+    const val CLEARING = "Clearing..."
+    const val CLEAR_LOG = "Clear log"
+    const val CLICK_TO_SELECT = "Click to select"
+    const val CLOSE_CONNECTION = "Close Connection"
+    const val CLOSE_TAB = "Close tab"
+    const val CLOSING = "Closing..."
+    const val COLLAPSE = "Collapse"
+    const val COMPILE = "Compile"
+    const val COMPILING = "Compiling..."
+    const val CONNECTION_CLOSED = "Connection closed"
+    const val COPY = "copy"
+    const val COPY_CURRENT_TAB = "Copy current tab"
+    const val COULD_NOT_TRANSMIT_TOPIC = "Could not transmit on topic"
+    const val DELETED_CLASS_FEEDBACK = "Selected class was in deleted file and is no longer available"
+    const val DELETE_JAR = "Delete jar"
+    const val ELLIPSIS = "…"
+    const val EXCEPTION = "Exception"
+    const val EXPAND = "Expand"
+    const val FAIL_TO_SUBSCRIBE = "Fail to subscribe"
+    const val FAILED_TO_CLOSE_PULSAR = "Failed to close pulsar"
+    const val FAILED_TO_CREATE_CONSUMER = "Failed to create consumer"
+    const val FAILED_TO_CREATE_DIRECTORY = "Failed to create directory"
+    const val FAILED_TO_CREATE_PRODUCER = "Failed to create producer"
+    const val FAILED_TO_DESERIALIZE_PROPERTIES = "Failed to deserialize properties"
+    const val FAILED_TO_DESERIALIZE_PULSAR = "Failed to deserialize pulsar"
+    const val FAILED_TO_LOAD_PROJECT = "Failed to load project"
+    const val FAILED_TO_RETRIEVE_TOPICS = "Failed to retrieve topics"
+    const val FAILED_TO_SEND_MESSAGE = "Failed to send message"
+    const val FAILED_TO_SETUP_PULSAR = "Failed to setup pulsar"
+    const val FILE = "File"
+    const val FILTER = "Filter"
+    const val GENERATE = "Generate"
+    const val GENERATED_CLASS_NOT_SAME_AS_SELECTED = "Generated class not the same as selected class"
+    const val GENERATED_CODE_TEMPLATE = "Generated code template"
+    const val GENERATING = "Generating..."
+    const val JARS = "Jars"
+    const val LOADING = "Loading..."
+    const val LOAD_PROJECT = "Load project"
+    const val LOAD_TOPICS = "Load topics"
+    const val MANAGE_CONFIGURATION_JARS = "Manage configurations jars"
+    const val MESSAGE = "Message"
+    const val MESSAGE_SENT_ID = "Message sent with id"
+    const val NAME = "Name"
+    const val NEW_TAB = "New tab"
+    const val NO_CLASS_GENERATED_TO_SEND = "No class generated to send"
+    const val NO_CLASS_SELECTED = "No class selected"
+    const val NO_CLASS_SELECTED_DESERIALIZE = "No class selected to deserialize"
+    const val NO_VALID_CLASSES_LOADED = "No valid classes loaded"
+    const val NO_TOPICS_FOUND = "No topics found"
+    const val ON_TOPIC = "on topic"
```

Comment:
```suggestion
    const val ON_TOPIC = "On topic"
```

Reply:
It's used as part of a sentence at the moment